### ISSUE TITLE
Is NodeTree supported when running with Qt?

### ISSUE THREAD

**Issue by itziakos** [enthought/pyface#120]
Importing NodeTree when using Qt results in the following tracebackthon

``` python
>>> from pyface.tree.api import NodeTree
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "c:\runtimes\simphony\lib\site-packages\pyface\tree\api.py", line 18, in <module>
    from node_tree import NodeTree
  File "c:\runtimes\simphony\lib\site-packages\pyface\tree\node_tree.py", line 28, in <module>
    from tree import Tree
  File "c:\runtimes\simphony\lib\site-packages\pyface\tree\tree.py", line 22, in <module>
    import wx
ImportError: No module named wx
```

The problem is that the node_tree module is tring to import the tree class from wx. So at first glance it looks that the NodeTree class does not support Qt. However,  I have labeled this issue as a bug because in https://github.com/enthought/pyface/blob/master/pyface/tree/api.py#L25 there is a specific check on the ets toolkit which means that this module should have support for Qt. I guess that the easy solution would be to import NodeTree only when wx is used.

**Comment by itziakos** [comment]
Note I discovered this issued when using the released versions of pyface and mayavi while running a python script that invokes the mayavi2 application

**Comment by corranwebster** [comment]
Fixed by #261.
